source/clear-linux/get-started/virtual-machine-install/kvm.rst

```diff
@@ -123,20 +112,20 @@ launched from, follow these steps.
         PermitRootLogin yes
         EOF
 
-#. Start SSH server in the |CL| VM:
+#. Enable and start SSH server in the |CL| VM:
 
    .. code-block:: bash
 
+      systemctl enable sshd
       systemctl start sshd
 
-#. From the host, SSH into the |CL| VM.  The port number ``10022`` is defined
-   in the ``start_qemu.sh`` script.
+#. SSH into the |CL| VM using the default port of  `10022`:
 
    .. code-block:: bash
 
-      ssh -p 10022 root@localhost
+      ssh -p 10022 root@<ip-addr-of-kvm-host>
```

Comment:
I run a simple command `ifconfig` to view it. However, if I use the value of the `inet addr`, it fails.
If, on the other hand, I use localhost, 127.0.0.1, it works. Why not clarify, specify your intent to the user? 
Provide the command to find the inet addr, or state use the localhost. I'm not convinced that you intend for the user to use 127.0.0.1 in this case, esp. since it's remote.

Reply:
@MCamp859 - Can you please help me add the following prior to this line?

"Determine the IP address of your host that you will be launching the VM on.  Substitute `<ip-addr-of-kvm-host>` in the next step with this information."

Also, add a code-block with:

$ ip a

Reply:
Done, with minor text wordsmithing and removal of the $ prompt.